index.html

```diff
@@ -1 +1,2 @@
 <h1>Hello World!</h1>
+<h1>Olá, Mundo!</h1>
```

Comment:
Could you remove the `Olá` string and make the `Mundo` string red?

Reply:
Sure, can do!